templates/forum/topic/index.html

```diff
@@ -269,17 +269,16 @@ <h3>{% trans "Développement" %}</h3>
                         <form action="{% url 'manage-issue' topic.pk %}" method="post" id="create-issue" class="modal modal-flex">
                             {% if has_token %}
                                 {% csrf_token %}
+                                <select name="repository">
+                                    {% for repo in repositories %}
+                                        <option value="{{repo.name}}">{{repo.name}}</option>
```

Comment:
Chipotage esthétique
```suggestion
                                        <option value="{{ repo.name }}">{{ repo.name }}</option>
```